src/containers/Chat/ChatConversations/ConversationList/ConversationList.tsx

```diff
@@ -212,30 +241,11 @@ export const ConversationList: React.SFC<ConversationListProps> = (props) => {
         </div>
       );
       conversationsData = conversations[dataArray].map((conversation: any, index: number) => {
-        let lastMessage = [];
-        lastMessage = conversation;
-        return (
-          <>
-            {index === 0 ? header : null}
-            <ChatConversation
-              key={conversation.contact.id}
-              selected={props.selectedContactId === conversation.contact.id}
-              onClick={() => props.setSelectedContactId(conversation.contact.id)}
-              index={index}
-              contactId={conversation.contact.id}
-              contactName={
-                conversation.contact.name ? conversation.contact.name : conversation.contact.phone
-              }
-              lastMessage={lastMessage}
-              senderLastMessage={conversation.contact.lastMessageAt}
-              contactStatus={conversation.contact.status}
-              contactBspStatus={conversation.contact.bspStatus}
-              highlightSearch={props.searchVal}
-              messageNumber={conversation.messageNumber}
-              searchMode={props.searchMode}
-            />
-          </>
-        );
+        // We don't have the last message in the case of contacts. Therefore, view contact only
+        if (dataArray === 'contacts') {
```

Comment:
@DigneshGujarathi This condition may not be needed now :) as you are handling this in buildChatConversation()